src/system/ExtensionsModule/Entity/Repository/ExtensionVarRepository.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Copyright Zikula Foundation 2015 - Zikula Application Framework
+ *
+ * This work is contributed to the Zikula Foundation under one or more
+ * Contributor Agreements and licensed to You under the following license:
+ *
+ * @license GNU/LGPLv3 (or at your option, any later version).
+ * @package Zikula_Form
+ *
+ * Please see the NOTICE file distributed with this source code for further
+ * information regarding copyright and licensing.
+ */
+
+namespace Zikula\ExtensionsModule\Entity\Repository;
+
+use Doctrine\ORM\EntityRepository;
+use Zikula\ExtensionsModule\Entity\RepositoryInterface\ExtensionVarRepositoryInterface;
+
+class ExtensionVarRepository extends EntityRepository implements ExtensionVarRepositoryInterface
+{
+    public function remove($entity)
+    {
+        $this->_em->remove($entity);
+    }
+
+    public function save($entity)
```

Comment:
Maybe call it `persistAndFlush` instead, because it clarifies what it actually does. Otherwise one might think that calling `remove` will flush as well.